### PR TITLE
refacto with cron file

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

File: roles/orchestraCron/tasks/mail.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+  - name: affect mail to
+    shell: "echo MAILTO={{mail_to}} > {{cron_file}}"
```
Comment (nicolasThal) [inline roles/orchestraCron/tasks/mail.yml:3]: what does happen if you run the provisioning multiple time?

Comment (itkg-rboessel) [inline roles/orchestraCron/tasks/mail.yml:3]: >    override the content of the file
>>  add to the end of the file
So the file will be reset each time

Comment (nicolasThal) [inline roles/orchestraCron/tasks/mail.yml:3]: sorry, I didn't saw it...

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**